conformance/apis/v1alpha1/conformancereport.go

```diff
@@ -69,5 +69,8 @@ type Implementation struct {
 	// Github usernames (in the form of `@<username>`) or team names (in the
 	// form of `@<team>/<name>`), but when that's not possible it can be email
 	// addresses.
+	// It can also be a link to the issue creation page or support documentation for a project.
+	// Ideally this would be something like the issue creation page on a repository, but for projects
+	// without a publicly exposed repository a general support page URL can be provided.
```

Comment:
```suggestion
	// Rather than Github usernames or email addresses you can provide a URL to the relevant
	// support pages for the project. Ideally this would be something like the issue creation page
	// on a repository, but for projects without a publicly exposed repository a general support 
	// page URL can be provided.
```